### PR TITLE
Some Enum completion

### PR DESCRIPTION
**Pull request by coalaura**
A variety of enums that i found.

### CODE REVIEW

**Comment by AvarianKnight** [comment]
can you make these use \```c codeblocks instead of \```cpp

**Comment by coalaura** [comment]
> can you make these use ```c codeblocks instead of ```cpp

yeah no problem :)

File: GRAPHICS/SetScriptGfxDrawOrder.md [hunk 1]
```diff
@@ -16,6 +16,21 @@ GRAPHICS::DRAW_RECT(0.5, 0.5, 3.0, 3.0, v_4, v_5, v_6, a_0._f172, 0);
...
+    GFX_ORDER_BEFORE_HUD_PRIORITY_LOW = 0,
+    GFX_ORDER_BEFORE_HUD = 1,
+    GFX_ORDER_BEFORE_HUD_PRIORITY_HIGH = 2,
+    GFX_ORDER_AFTER_HUD_PRIORITY_LOW = 3,
+	GFX_ORDER_AFTER_HUD = 4,
+	GFX_ORDER_AFTER_HUD_PRIORITY_HIGH = 5,
+	GFX_ORDER_AFTER_FADE_PRIORITY_LOW = 6,
+	GFX_ORDER_AFTER_FADE = 7,
+	GFX_ORDER_AFTER_FADE_PRIORITY_HIGH = 8,
+}
```
Comment (colistro123) [inline GRAPHICS/SetScriptGfxDrawOrder.md:31]: May I suggest indenting this?

```suggestion
enum eGfxDrawOrder
{
    GFX_ORDER_BEFORE_HUD_PRIORITY_LOW = 0,
    GFX_ORDER_BEFORE_HUD = 1,
    GFX_ORDER_BEFORE_HUD_PRIORITY_HIGH = 2,
    GFX_ORDER_AFTER_HUD_PRIORITY_LOW = 3,
    GFX_ORDER_AFTER_HUD = 4,
    GFX_ORDER_AFTER_HUD_PRIORITY_HIGH = 5,
    GFX_ORDER_AFTER_FADE_PRIORITY_LOW = 6,
    GFX_ORDER_AFTER_FADE = 7,
    GFX_ORDER_AFTER_FADE_PRIORITY_HIGH = 8,
}
```

File: HUD/GetPauseMenuState.md [hunk 2]
```diff
@@ -8,16 +8,18 @@ ns: HUD
...
+{
+    PM_INACTIVE 	 =  0,
+	PM_STARTING_UP 	 =  5,
+	PM_RESTARTING 	 = 10,
+	PM_READY 		 = 15,
+	PM_IN_STORE 	 = 20,
+	PM_IN_SC_MENU 	 = 25,
+	PM_SHUTTING_DOWN = 30,
+	PM_IN_VIDEOEDITOR = 35,
+}
```
Comment (colistro123) [inline HUD/GetPauseMenuState.md:22]: Indent 😊

```suggestion
enum ePauseMenuState
{
    PM_INACTIVE 	 =  0,
    PM_STARTING_UP 	 =  5,
    PM_RESTARTING 	 = 10,
    PM_READY 		 = 15,
    PM_IN_STORE 	 = 20,
    PM_IN_SC_MENU 	 = 25,
    PM_SHUTTING_DOWN = 30,
    PM_IN_VIDEOEDITOR = 35,
}
```

File: HUD/SetTextFont.md [hunk 3]
```diff
@@ -8,8 +8,19 @@ ns: HUD
...
+    FONT_STANDARD = 0,
+    FONT_CURSIVE = 1,
+    FONT_ROCKSTAR_TAG = 2,
+    FONT_LEADERBOAR0D = 3,
+    FONT_CONDENSED = 4,
+    FONT_STYLE_FIXED_WIDTH_NUMBERS = 5,
+    FONT_CONDENSED_NOT_GAMERNAME = 6,
+	FONT_STYLE_PRICEDOWN = 7,
+	FONT_STYLE_TAXI = 8,
+}
```
Comment (colistro123) [inline HUD/SetTextFont.md:23]: This too.

```suggestion
enum eTextFonts
{
    FONT_STANDARD = 0,
    FONT_CURSIVE = 1,
    FONT_ROCKSTAR_TAG = 2,
    FONT_LEADERBOAR0D = 3,
    FONT_CONDENSED = 4,
    FONT_STYLE_FIXED_WIDTH_NUMBERS = 5,
    FONT_CONDENSED_NOT_GAMERNAME = 6,
    FONT_STYLE_PRICEDOWN = 7,
    FONT_STYLE_TAXI = 8,
}
```

File: MISC/SetGravityLevel.md [hunk 4]
```diff
@@ -8,12 +8,14 @@ ns: MISC
...
-2: 0.1 - very low  
-3: 0.0  
+```c
+enum eGravityLevel
+{
+    GRAV_EARTH = 0, // earth gravity 9.8m/s2
+    GRAV_MOON = 1, // moon gravity 2.4m/s2
+	GRAV_LOW = 2, // very low gravity
+    GRAV_ZERO = 3 // zero gravity
+}
```
Comment (colistro123) [inline MISC/SetGravityLevel.md:18]:
```suggestion
enum eGravityLevel
{
    GRAV_EARTH = 0, // earth gravity 9.8m/s2
    GRAV_MOON = 1, // moon gravity 2.4m/s2
    GRAV_LOW = 2, // very low gravity
    GRAV_ZERO = 3 // zero gravity
}
```

File: PED/ForcePedMotionState.md [hunk 5]
```diff
@@ -8,26 +8,28 @@ ns: PED
...
+    MS_DIVING_SWIM		= -1855028596, // motionstate_diving_swim
+    MS_PARACHUTING		= -1161760501, // motionstate_parachuting
+    MS_AIMING			= 1063765679,  // motionstate_aiming
+    MS_ACTIONMODE_IDLE	= -633298724,  // motionstate_actionmode_idle
+    MS_ACTIONMODE_WALK	= -762290521,  // motionstate_actionmode_walk
+    MS_ACTIONMODE_RUN	= 834330132,   // motionstate_actionmode_run
+    MS_STEALTHMODE_IDLE	= 1110276645,  // motionstate_stealth_idle
+    MS_STEALTHMODE_WALK	= 69908130,    // motionstate_stealth_walk
+    MS_STEALTHMODE_RUN	= -83133983,   // motionstate_stealth_run
+}
```
Comment (colistro123) [inline PED/ForcePedMotionState.md:32]:
```suggestion
enum ePedMotionState
{
    MS_ON_FOOT_IDLE = -1871534317, // motionstate_idle
    MS_ON_FOOT_WALK = -668482597, // motionstate_walk
    MS_ON_FOOT_RUN = -530524, // motionstate_run
    MS_ON_FOOT_SPRINT = -1115154469, // motionstate_sprint
    MS_CROUCH_IDLE = 1140525470, // motionstate_crouch_idle
    MS_CROUCH_WALK = 147004056, // motionstate_crouch_walk
    MS_CROUCH_RUN = 898879241, // motionstate_crouch_run
    MS_DO_NOTHING = 247561816, // motionstate_donothing
    MS_DIVING_IDLE = 1212730861, // motionstate_diving_idle
    MS_DIVING_SWIM = -1855028596, // motionstate_diving_swim
    MS_PARACHUTING = -1161760501, // motionstate_parachuting
    MS_AIMING = 1063765679, // motionstate_aiming
    MS_ACTIONMODE_IDLE = -633298724, // motionstate_actionmode_idle
    MS_ACTIONMODE_WALK = -762290521, // motionstate_actionmode_walk
    MS_ACTIONMODE_RUN = 834330132, // motionstate_actionmode_run
    MS_STEALTHMODE_IDLE = 1110276645, // motionstate_stealth_idle
    MS_STEALTHMODE_WALK = 69908130, // motionstate_stealth_walk
    MS_STEALTHMODE_RUN = -83133983, // motionstate_stealth_run
}
```

**Review by colistro123 — COMMENTED** [review]
Suggested on indenting a couple things 😊

**Pull request merged**